src/components/menu/MenuSeparator.tsx

```diff
@@ -1,5 +1,12 @@
 import React from 'react';
+import classNames from 'classnames';
 
-const MenuSeparator = () => <li role="separator" />;
+type MenuSeparatorProps = {
+    className?: string;
+};
+
+const MenuSeparator = ({ className }: MenuSeparatorProps) => (
+    <li className={classNames('MenuSeparator', className)} role="separator" />
```

Comment:
Let's add a `bdl-` prefix :)
```suggestion
    <li className={classNames('bdl-MenuSeparator', className)} role="separator" />
```